### PR TITLE
Vagov 1330 trigger information must be a button keyboard focusable

### PR DESCRIPTION
**Pull request by mojaray2k**
## Description

The `<a>` tags to open the modal for entering direct deposit information are not keyboard focusable. They should be `<button>` tags and have some specific requirements for keyboard navigation. Screenshot attached below.
## Testing done
Was able to use the keyboard to open and close the modal and stay on the same clickable button
## Screenshots
Your_Profile___Veterans_Affairs_and_Updating_Docker

## Acceptance criteria
- [ ]

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/personalization/profile360/containers/PaymentInformation.jsx [hunk 1]
```diff
@@ -177,39 +177,39 @@ class PaymentInformation extends React.Component {
           <div className="vet360-profile-field">
             <ProfileFieldHeading
               onEditClick={
-                directDepositIsSetUp &&
+                !directDepositIsSetUp &&
```
Comment (erikphansen) [inline src/applications/personalization/profile360/containers/PaymentInformation.jsx]: gotta take all these `!` changes out

File: src/applications/personalization/profile360/sass/profile-360.scss [hunk 2]
```diff
@@ -70,3 +70,13 @@ input.usa-only-phone {
     margin: 0;
   }
 }
+[type="submit"].vads-u-width--auto {
```
Comment (erikphansen) [inline src/applications/personalization/profile360/sass/profile-360.scss]: this seems leftover from an older branch you were working on.

**Review by erikphansen — CHANGES_REQUESTED** [review]

File: src/applications/personalization/profile360/sass/profile-360.scss (hunk 2 above)
Comment (mojaray2k) [reply src/applications/personalization/profile360/sass/profile-360.scss]: Code is removed

File: src/applications/personalization/profile360/containers/PaymentInformation.jsx (hunk 1 above)
Comment (mojaray2k) [reply src/applications/personalization/profile360/containers/PaymentInformation.jsx]: I changed them back.

File: src/applications/personalization/profile360/containers/PaymentInformation.jsx [hunk 3]
```diff
@@ -143,9 +143,9 @@ class PaymentInformation extends React.Component {
 
   renderSetupButton(label, gaProfileSection) {
     return (
-      <a
+      <button className="va-button-link"
```
Comment (erikphansen) [inline src/applications/personalization/profile360/containers/PaymentInformation.jsx]: ESLint is gonna complain about this line. You can see this by running `yarn lint:js`. You can also set up VSCode to use the ESLint extension and show (and sometimes fix) the errors right in the editor.

Comment (mojaray2k) [reply src/applications/personalization/profile360/containers/PaymentInformation.jsx]: Just a straight brain fart. I didn't have any vscode extensions installed. I just installed React Food Truck and was able to start seeing eslint errors.

File: yarn.lock [hunk 4]
```diff
@@ -8808,6 +8808,7 @@ metalsmith-date-in-filename@^0.0.14:
 metalsmith-filenames@^1.0.0:
   version "1.0.0"
   resolved "https://registry.yarnpkg.com/metalsmith-filenames/-/metalsmith-filenames-1.0.0.tgz#df4e69f246b8325bb9189e0273d2ee70899d00de"
+  integrity sha1-305p8ka4Mlu5GJ4Cc9LucImdAN4=
```
Comment (erikphansen) [inline yarn.lock]: I'm still puzzled by how/why this was introduced but I think it should be fine.

**Review by erikphansen — APPROVED** [review]

File: yarn.lock (hunk 4 above)
Comment (mojaray2k) [reply yarn.lock]: Yeah and the scripts commands for linting are not working for me either.

**Review by rianfowler — APPROVED** [review]

**Pull request merged**